experimental/tiledb/common/dag/nodes/doc/api.md

```diff
@@ -0,0 +1,456 @@
+
```

Comment:
@NikolaosPapailiou please take a look here, in case you haven't seen this doc yet.